pyproject.toml

```diff
@@ -21,3 +21,14 @@ pylint = "*"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[project]
+dependencies = [
+   "python == ^3.7",
+   "jsonschema == *",
+   "pyyaml == *",
+   "cbor2 == *",
+   "Jinja2 == *",
+   "stringcase == 1.2.0",
+   "GitPython == 3.1.18"
+]
```

Comment:
What's going on here? This looks redundant with `[tool.poetry.dependencies]`.

Reply:
Nope. `tool.poetry` is poetry. `project` is pip (or setuptools). I agree it's weird that they are in the same .toml file.

Reply:
Hang on a bit; poetry seems to be processing the `project` part as well, which we probably don't want. I didn't see it because poetry is still misbehaving on my Mac, but when the CI doesn't use a cached env, I'm seeing a line under dependencies I probably shouldn't. (Or maybe it doesn't matter, but need to investigate.)